orttraining/orttraining/test/python/checkpoint/_test_helpers.py

```diff
@@ -103,7 +105,7 @@ def create_orttrainer_and_load_checkpoint(device, trainer_opts, checkpoint_dir,
 def split_state_dict(state_dict):
     """Given a flat state dictionary, split it into optimizer, fp32_param, fp16_param hierarchical dictionary and return"""
 
-    optimizer_keys = ['Moment_1_', 'Moment_2_', 'Update_Count_', 'Step_']
+    optimizer_keys = ['Moment_1_', 'Moment_2_', 'Update_Count_', 'Step']
```

Comment:
Helpers like https://github.com/microsoft/onnxruntime/pull/6136/files#diff-24a6f041f9a0a6b6544047e3a746ea9650adc0babea000148d8b975ebc3daba8R182 could remove this kind of hard-coding and prevent duplication. Maybe we can refactor this part when #6136 is merged

Reply:
sounds good